src/tools/pmempool/pmempool.c

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: BSD-3-Clause
-/* Copyright 2014-2022, Intel Corporation */
+/* Copyright 2014-2023, Intel Corporation */
 
 /*
  * pmempool.c -- pmempool main source file
```

Comment:
Using the blk function still does not display deprecated messages. The main help also does not provide this information.